core/assets/js/app.js

```diff
@@ -118,19 +130,20 @@ Hooks.PythonUploader = {
     })
   }
 }
-let liveSocket = new LiveSocket("/live", Socket, {
-  params: {
-    _csrf_token: csrfToken,
-  },
+
+let liveSocket = new LiveSocket('/live', Socket, {
   dom: {
     onBeforeElUpdated(from, to) {
       if (from.__x) {
-        Alpine.clone(from.__x, to);
+        window.Alpine.clone(from.__x, to)
```

Comment:
Why this change?

Reply:
A left over from some changes I tried to fix an Alpine data state problem of first time after load. I think it is safe to remove